CMakeLists.txt

```diff
@@ -124,4 +124,10 @@ if (SPIRV_HEADERS_ENABLE_INSTALL)
         NAMESPACE "${namespace}"
         DESTINATION "${config_install_dir}"
     )
+
+    configure_file(${CMAKE_SOURCE_DIR}/SPIRV-Headers.pc.in ${CMAKE_BINARY_DIR}/SPIRV-Headers.pc @ONLY)
```

Comment:
Currently, breaks shaderc compilation with
```cmake
-- Installing SPIRV-Header
CMake Error: File E:/media-autobuild_suite-master/build/shaderc-git/SPIRV-Headers.pc.in does not exist.
CMake Error at third_party/spirv-headers/CMakeLists.txt:128 (configure_file):
  configure_file Problem configuring file
```

Possible solution is
```diff
-    configure_file(${CMAKE_SOURCE_DIR}/SPIRV-Headers.pc.in ${CMAKE_BINARY_DIR}/SPIRV-Headers.pc @ONLY)
+    configure_file(${CMAKE_CURRENT_SOURCE_DIR}/SPIRV-Headers.pc.in ${CMAKE_BINARY_DIR}/SPIRV-Headers.pc @ONLY)
```
or to use `PROJECT_SOURCE_DIR` to ensure it always points to the correct file

Reply:
It doesn't help as this file keeps on reseting to its faulty state...